### ISSUE TITLE
Linking users

### ISSUE THREAD

**Issue by ahnl** [Testausserveri/testausserveri.fi#36]
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/46541386/167254325-faca2ce1-f550-4905-b900-66a74bec8477.png">

Discord and Testaustime has different usernames, but maybe a fun idea would be to somehow unify them in our website. This would allow us to show profile pictures on all names and also make names clickable: click on a name and you'll get to member's profile page (in the Jäsenet section).

One way could be to let members specify themselves their Discord user id and Testaustime name on our platform. In this case we may want to wait until we implement the Jäsensivut (member site).

**Comment by kymppi** [comment]
Testausserveri-id?

**Comment by ahnl** [comment]
> Testausserveri-id?

Yes, we are planning to use Testausserveri-id eventually on the platform (for member site).
